CONTRIBUTING.md

```diff
@@ -210,4 +223,4 @@ nothing.
 #### Step 7: Deploy updates to the `master` branch
 If the `master` branch has been updated, you can push all the changes
 to the dash-bio remote. However, make sure that the `Procfile` on the
```

Comment:
```suggestion
to your `origin` remote. However, make sure that the `Procfile` on the
```
I guess that was the intended meaning... `dash-bio` would not be a great name for a remote, since it's already the repo's name.